src/cegpy/graphs/_ceg.py

```diff
@@ -190,11 +190,9 @@ def _merge_nodes(self, nodes_to_merge: List[Tuple[str]]):
 
     def dot_graph(self, edge_info: str = "probability") -> pdp.Dot:
```

Comment:
```suggestion
    def dot_graph(self, edge_info: Literal["count", "prior", "posterior", "probability"] | None = "probability") -> pdp.Dot:
```

`Literal["count", "prior", "posterior", "probability"]` -- this says that only "count", "prior", "posterior" or "probability" are acceptable input parameters. 

 `|` -- this means "or"

`None` -- None is its own type in Python, and so we shouldn't be using `"none"` as a string. 

With the above setting, the user will need to explicitly pass `None` for no information to be displayed. 
Alternatively, we can make `None` the default, so the user needs to explicitly pass one of the acceptable string entries for any information to be displayed. 

What do you think?